repositories/cyclonedds.BUILD.bazel

```diff
@@ -99,7 +107,7 @@ cache_entries_linux_and_macos = {
 cache_entries_qnx = {
     "ENABLE_SOURCE_SPECIFIC_MULTICAST": "OFF",
     "ENABLE_IPV6": "OFF",
-    "CMAKE_SYSTEM_NAME": "QNX",
+    "CMAKE_SYSTEM_NAME": "QNX_OR_ANDROID",
```

Comment:
Is it good to call it `QNX_OR_ANDROID` & `cache_entries_qnx`?

Reply:
Should we add CI for different platforms also? @mvukov

Reply:
The naming is fine. Thanks for the contribution. BTW, can you share how would you use ROS+android?

Regarding CI: ATM I dunno how would we set up CI for Android. IMO, this PR would be good to go as-is, once the CI is green.

Reply:
Well, compilation doesn't mean that e.g. a deployment works. Do Python-based ROS 2 deployments (ros2_launch targets) work on Android for you? IMO, this is a bit more involved issue -- we can leave this for a follow-up.